docs/useNotify.md

```diff
@@ -216,4 +221,47 @@ export const ConnectionWatcher = () => {
 };
 ```
 
-Note that if you use this ability to pass a React node, the message will not be translated - you'll have to translate it yourself using [`useTranslate`](./useTranslate.md).
\ No newline at end of file
+Note that if you use this ability to pass a React node, the message will not be translated - you'll have to translate it yourself using [`useTranslate`](./useTranslate.md).
+
+If you have custom actions in your notification element, you can leverage the `useCloseNotification` hook to close the notification:
+
+```tsx
+import { useFormContext } from 'react-hook-form';
+import { Button, useCloseNotification, useNotify } from 'react-admin';
+import { SnackbarContent } from '@mui/material';
+
+const SetFormValueButton = () => {
+    const { setValue } = useFormContext();
+    const notify = useNotify();
+
+    return (
+        <Button
+            onClick={() => {
+                setValue('myfield', 'a value');
+                notify(<SetFormValueNotification />);
+            }}
+            label="Set myfield value"
+        />
+    );
+};
+
+const SetFormValueNotification = () => {
+    const { setValue } = useFormContext();
```

Comment:
I think the `setValue` callback should be injected by the button. Otherwise, how can we be sure the notification is rendered inside the form context?